spec/lucky/input_helpers_spec.cr

```diff
@@ -91,6 +91,9 @@ describe Lucky::InputHelpers do
       view.checkbox(false_field).to_s.should contain <<-HTML
       <input type="hidden" id="" name="user:admin" value="false">
       HTML
+      view.checkbox(false_field, attrs: [:checked]).to_s.should contain <<-HTML
+			<input type="checkbox" id="user_admin" name="user:admin" value="true" checked>
```

Comment:
local version doesn't have this space, and crystal tool format doesn't fix it.... 

<img width="195" alt="Screen Shot 2019-10-22 at 8 53 12 PM" src="https://user-images.githubusercontent.com/2391/67356112-fa0c4980-f50d-11e9-9aba-f0931119fd9b.png">

Reply:
turned out I didn't have tabs turned off.